tests/search/search_export_tests.py

```diff
@@ -0,0 +1,69 @@
+import json
+from django.test import Client
+from django.urls import reverse
+from arches.app.models import models
+from django.contrib.auth.models import User
+from arches.app.search.search_export import SearchResultsExporter
+from django.test.client import RequestFactory
+from tests.base_test import ArchesTestCase
+
+# these tests can be run from the command line via
+# python manage.py test tests.search.search_export_tests --settings="tests.test_settings"
+
+class SearchExportTests(ArchesTestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.factory = RequestFactory()
+        cls.client = Client()
```

Comment:
TIL `SimpleTestCase` already comes with a client.

```suggestion
```

Reply:
fix in latest